go/vt/sqlparser/ast.go

```diff
@@ -1570,6 +1584,110 @@ func (node *ValuesFuncExpr) WalkSubtree(visit Visit) error {
 	)
 }
 
+// ConvertExpr represents a call to CONVERT(expr, type)
+// CONVERT(expr USING transcoding_name) it not supported
+type ConvertExpr struct {
+	Expr Expr
+	Type *ConvertType
+}
+
+// Format formats the node.
+func (node *ConvertExpr) Format(buf *TrackedBuffer) {
+	buf.Myprintf("convert( ")
+	if node.Expr != nil {
+		buf.Myprintf("%v", node.Expr)
+	}
+	buf.Myprintf(" , ")
```

Comment:
should this line be in the above if statement? Otherwise if Expr is nil, we will see something like `convert( , node.Type)` or `convert( ,)` if Type is also nil.

Also, is it possible in the grammar for these to be nil?